source/style-guide/style/placeholder-variable-text.txt

```diff
@@ -26,13 +28,79 @@ name of a directory.
    the path. For information about formatting environment variables,
    see :ref:`text-formatting`.
 
+Placeholder Text Guidelines
+---------------------------
+
 When creating placeholder text, use the following guidelines.
 
+Apply Language-Specific Formatting
```

Comment:
One issue I've already run into is that the drivers teams do not necessarily use any format linting tools and also do not adhere to these standards. We started a document on standardizing the formatting across the DevEd team in this doc https://docs.google.com/spreadsheets/d/1vxDasa0S2yrpXfTJkH8VooYBIasiHdOIotQOx_b8xfc/edit but not sure if it's currently being followed. However, when we call each other out on formatting errors, these are our standards.

Reply:
We're similar in this doc, actually. I don't think there is a huge gap. JavaScript is missing, here, though.

Reply:
We'll probably need to communicate this to drivers teams as we work on more documentation for them. My guess is that they would be ok with them, but would want to be made aware.